assets/js/components/ActivateModuleCTA.js

```diff
@@ -58,37 +59,36 @@ const ActivateModuleCTA = ( { slug, title, description } ) => {
 				type: 'win-error',
 			} );
 		}
-	} );
+	}, [ activateModule ] );
```

Comment:
`activateModule` comes from `useDispatch` here which is generally stable, but could potentially return different values due to async store registration, so good to be a little extra safe here.